### ISSUE TITLE
Community chat server? (off-topic)

### ISSUE THREAD

**Issue by JasminDreasond** [NodeL2/NodeL2#6]
I would recommend a Discord server or Matrix server. A place where we could have better contact to talk about the subject. I will usually be available Saturday or Sunday to participate in hobby projects.

**Comment by dkoluris** [comment]
Hi Jasmin! I appreciate the collaboration proposal, but unfortunately I am not active, nor have a lot of free time for this project. Nonetheless, I can be available to answer everything related to it via GitHub, Email or Discord in a private channel.

**Comment by JasminDreasond** [comment]
Yes. I accept contact via DIscord. Feel free to share your Discord username.

I've already started studying the repository and made a fork so I can start having a little fun during my free coding times. :p

This game is very nostalgic for me, and combining that with nodejs which is something I really enjoy working on makes things more fun.

**Comment by dkoluris** [comment]
Yup, nostalgic for me as well. I love NodeJS too. Also, I play from time to time on some private servers for the kicks. Discord username is `.denicio`.
